run_trigger.go

```diff
@@ -57,7 +57,7 @@ type RunTrigger struct {
 	SourceableName string    `jsonapi:"attr,sourceable-name"`
 	WorkspaceName  string    `jsonapi:"attr,workspace-name"`
 	// DEPRECATED. The sourceable field is polymorphic. Use SourceableChoice instead.
-	Sourceable       *Workspace        `jsonapi:"relation,sourceable"`
+	Sourceable       *Workspace        `jsonapi:"relation,sourceable,omitempty"`
```

Comment:
This is the only other example of a jsonapi `relation` being superseded by `polyrelation`. With the new jsonapi changes, we have to make sure that we're manually setting the deprecated field.